state-chain/chains/src/dot.rs

```diff
@@ -1068,4 +1140,25 @@ mod test_polkadot_extrinsics {
 			extrinsic_builder.get_signed_unchecked_extrinsic().unwrap().encode()
 		);
 	}
+
+	#[test]
+	fn fee_estimation_doesnt_overflow() {
+		let ingress_fee = PolkadotTrackedData {
+			median_tip: Default::default(),
+			runtime_version: Default::default(),
+		}
+		.estimate_ingress_fee(assets::dot::Asset::Dot);
+
+		let egress_fee = PolkadotTrackedData {
+			median_tip: Default::default(),
+			runtime_version: Default::default(),
+		}
+		.estimate_egress_fee(assets::dot::Asset::Dot);
+
+		// The values are not important. This test serves more as a sanity check that
+		// the fees are valid, and a reference to compare against the actual fees. These values must
+		// be updated if we update the fee calculation.
+		assert_eq!(ingress_fee, 21004500u128);
+		assert_eq!(egress_fee, 24004500u128);
```

Comment:
I need to check again that these fees are reasonable. I think it might still be off by a factor of ~10...

For reference, a regular balance transfer on Polkadot costs 0.0158 DOT, which is 158_000_000 units.